index.js

```diff
@@ -1,17 +1,46 @@
 class SortedList {
-  constructor() {}
 
-  add(item) {}
+  //this.items = [];
 
-  get(pos) {}
+  constructor(){
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  max() {}
+  add(item){
+    this.items.unshift(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  min() {}
+  get(pos){
+    if(pos > 0 && pos < this.length){
```

Comment:
Aqui vc pode fazer uma verificação mais simples, do próprio this.items[pos]

`
if (this.items[pos]) {
    return this.items[pos]
}

throw new Error('OutOfBounds');
`